ark/utils/data_utils_test.py

```diff
@@ -328,14 +227,8 @@ def test_generate_deepcell_input():
         fov1 = io.imread(fov1path)
         fov2 = io.imread(fov2path)
 
-        # check shape
-        assert fov1.shape == (1024, 1024, 2)
-        assert fov2.shape == (1024, 1024, 2)
-
-        assert np.all(fov1[:, :, 0] == 2)
-        assert np.all(fov1[:, :, 1] == 3)
-        assert np.all(fov2[:, :, 0] == 1)
-        assert np.all(fov2[:, :, 1] == 1)
+        assert np.all(fov1 == data_xr.loc['fov1', :, :, ['nuc2', 'mem2']].values)
```

Comment:
I think using np.array_equal would be more clear here to indicate we're comparing values